.sonarcloud.properties

```diff
@@ -1,2 +1,2 @@
 # Disable specific duplicate code since it would introduce more complexity to reduce it.
-sonar.cpd.exclusions=src/generators/javascript/renderers/ClassRenderer.ts,src/generators/typescript/renderers/ClassRenderer.ts,src/generators/javascript/JavaScriptRenderer.ts,src/generators/java/renderers/ClassRenderer.ts,src/generators/typescript/TypeScriptRenderer.ts,src/generators/typescript/renderers/EnumRenderer.ts,src/generators/java/renderers/EnumRenderer.ts
+sonar.cpd.exclusions=src/generators/**/**/*.ts
```

Comment:
```suggestion
sonar.cpd.exclusions=src/generators/**/*.ts
```

Is not enough?

Reply:
Not sure if it will catch `src/generators/javascript/renderers/ClassRenderer.ts` 🤔 It depends on how they resolve the path...

Reply:
It config is correct it's even propagated to PR, so you can see if any errors pop up :)

Reply:
Right, lets see then :D